packages/angular_devkit/build_angular/src/builders/karma/tests/options/include_spec.ts

```diff
@@ -17,16 +17,8 @@ describeBuilder(execute, KARMA_BUILDER_INFO, (harness) => {
         include: ['abc.spec.ts', 'def.spec.ts'],
       });
 
-      const { result, logs } = await harness.executeOnce();
+      const { result } = await harness.executeOnce();
       expect(result?.success).toBeFalse();
-      expect(logs).toContain(
```

Comment:
Shouldn't the log message should still be here even with the logic change?

Reply:
Form my understanding Webpack will print the logs to the console directly.